### PR TITLE
Update to svelte 5

### PR DESCRIPTION
**Pull request by jdromero88**
Update to use Svelte 5.

To test and run this smoothly, make sure you are using node 22. The exact one I'm using is `node v22.10.0 (npm v10.9.0)`

You can test this branch here: https://update-to-svelte-5--stateenergy.netlify.app/
and after is merged it can be tested in the staging branch https://staging--stateenergy.netlify.app/

### CODE REVIEW

File: src/lib/Table.svelte [hunk 1]
```diff
@@ -11,29 +11,32 @@
 
   const sortByColumns = ['activity', 'state', 'authority', 'type of resource']
 
-  function handleClick(e) {
+function handleClick(e) {
+  // console.log(e)
```
Comment (urchykli) [inline src/lib/Table.svelte:15]: Let's delete this commented out console log.

File: src/lib/Table.svelte [hunk 2]
```diff
@@ -11,29 +11,32 @@
...
     title.classList.toggle('table__body__cell--border')
     currentRow.classList.toggle('table__body__cell--border')
     // Show/Hide extraContent
-    extraContent.classList.toggle('active')
-    extraContent.classList.toggle("hide")
+    extraContent.classList.toggle('active');
+    extraContent.classList.toggle("hide");
     (row.isOpen) ? row.isOpen = true : row.isOpen = !row.isOpen
-  }
+    // row.isOpen = row.isOpen ? true : !row.isOpen
```
Comment (urchykli) [inline src/lib/Table.svelte:37]: If we don't need this let's delete it.

File: src/scss/components/_table.scss [hunk 3]
```diff
@@ -183,7 +183,8 @@
       padding: 0;
       width: calc(100vw - rem(32) * 2);
       max-width: rem(750);
-      max-height: 0px;
+      // max-height: 0px;
```
Comment (urchykli) [inline src/scss/components/_table.scss:186]: If we don't need this let's delete it.

File: src/scss/components/_table.scss [hunk 4]
```diff
@@ -411,17 +411,15 @@
...
-    transition: $transition__link;
-  }
+// .title--active .table__body__cell__title-container .icon-container::before {
+//   transform: rotate(-45deg) !important;
+//   transition: $transition__link !important;
+// }
+// .title--active .table__body__cell__title-container .icon-container::after {
+//   transform: rotate(45deg) !important;
+//   transition: $transition__link !important;
+// }
```
Comment (urchykli) [inline src/scss/components/_table.scss:421]: If we don't need this let's delete it.

**Review by urchykli — CHANGES_REQUESTED** [review]
Great work on this, Jose! In addition to the things I noted to delete, if we are no longer using Rollup, let's remove it from the README. Thanks for your hard work on this!

**Comment by jdromero88** [comment]
I removed all the commented lines and add a function to sort the dropdowns alphabetically.

File: src/scss/components/_table.scss (hunk 3 above)
Comment (urchykli) [reply src/scss/components/_table.scss:186]: This commented out code is still in the file - do we need it?

**Review by urchykli — CHANGES_REQUESTED** [review]
Thanks for cleaning this up, Jose! There's one more commented out style declaration that needs to be deleted and then you can merge!

**Pull request merged**